.circleci/config.yml

```diff
@@ -2,19 +2,19 @@ version: 2
 jobs:
   lint:
     docker:
-      - image: golangci/golangci-lint:v1.45-alpine
+      - image: golangci/golangci-lint:v1.55-alpine
     steps:
       - checkout
       - run: golangci-lint run
 
   # The kafka 0.10 tests are maintained as a separate configuration because
   # kafka only supported plain text SASL in this version.
   kafka-010:
-    working_directory: &working_directory /go/src/github.com/segmentio/kafka-go
+    working_directory: &working_directory /home/circleci/src/github.com/segmentio/kafka-go
     environment:
       KAFKA_VERSION: "0.10.1"
     docker:
-    - image: circleci/golang
+    - image: cimg/go:1.21
```

Comment:
I think it may be best to test against the same Go version that is in our `go.mod` file